tests/integration/test_store.py

```diff
@@ -72,8 +72,11 @@ def test_options_response(mini_sentry, relay, method_to_test):
     }
 
     result = relay.send_options(project_id, headers)
-
-    assert result.ok == should_succeed
+    assert result.ok, result
```

Comment:
The previous test asserted invalid behavior. The preflight should always succeed, its response headers declare whether the stated request would be valid.